services/src/main/scala/cromwell/services/metadata/impl/MetadataServiceActor.scala

```diff
@@ -161,6 +177,16 @@ case class MetadataServiceActor(serviceConfig: Config, globalConfig: Config, ser
     }
   }
 
+  private def sendMetadataTableSizeMetrics(): Unit = {
+    def convertToGiB(bytes: Long): Long = bytes / (1024 * 1024 * 1024)
```

Comment:
I think it's better to leave these as raw numbers and tell grafana that the unit is "bytes". This happens in the `Bytes Processed` metric at the top of the Cromwell Archiving page, and grafana automatically converts from the raw bytes into the most appropriate human-readable unit.

Reply:
TIL. If that's the case then yeah I agree it would be best to leave it as raw numbers. I will change it. Thank you!